wgsl/index.bs

```diff
@@ -5238,8 +5246,12 @@ An expression is an override-expression if all its [=identifiers=] [=resolve=] t
 
 Note: All [=const-expressions=] are also override-expressions.
 
+Override-expressions are only evaluated after any API-provided values are
+substituted [=override-declarations=].
+If an [=override-declaration=] has its value substituted via the API, its
+initializer expression, if present, will not be evaluated.
 An override-expression |E| [=behavioral requirement|will=] be evaluated if and only if:
```

Comment:
Suggest starting this with "Otherwise, an override-expression..."  to make it clear this is paired up with the previous sentence.